fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraBinaryImpl.java

```diff
@@ -270,14 +276,20 @@ public String getFilename() {
     }
 
     @Override
-    public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator) {
-        return getFixity(idTranslator, getContentDigest(), getContentSize());
+    public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> idTranslator,
+                               final String baseURL,
+                               final String userAgent,
+                               final EventBus eventBus) {
```

Comment:
@awoods These kinds of shenanigans in parameter lists are why we should think about bringing the core model classes into container management. I had to do something similar with the blank nodes thing.